srcpkgs/QSyncthingTray/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'QSyncthingTray'
+pkgname=QSyncthingTray
+version=0.5.8
+revision=1
+build_style=cmake
+makedepends="qt5-declarative-devel qt5-webengine-devel qt5-webchannel-devel qt5-location-devel qt5-quickcontrols2-devel qt5-devel"
+short_desc="A OS X / Windows / Linux Tray App for Syncthing written in C++"
```

Comment:
Imo `Tray App for Syncthing written in C++` would be enough. Also never Start short_desc with a `A`

Reply:
Fair enough - I simply copied over the repo description, but I guess that works as well.